src/base/bittorrent/torrentimpl.cpp

```diff
@@ -554,6 +555,26 @@ QVector<TrackerEntry> TorrentImpl::trackers() const
     return entries;
 }
 
+bool TorrentImpl::hasWorkingTracker() const
+{
+    const auto currentTrackers = asConst(trackers());
```

Comment:
No need for `asConst()` here. Only use it in for loops when the container is not const and only for Qt types (QVector, QList etc).